edkahara/iReporter
# [finishes #162527329] fix api v1 bugs

__What does this PR do?__
Fix bugs from api v1.

__Description of tasks to be completed.__
- Configure the application better
- Ensure that a user cannot register twice
- Ensure that each user and each report have unique id's
- Ensure that the responses for POST and PATCH return the incidents and/or users that have been posted and patched respectively
- Ensure that the report type can only be Red-Flag or Intervention

__How should it be manually tested?__
Clone the project. Check out the bg-fix-api-v1-bugs-#162527329 branch. Start the environment using env.bat. Run 'flask run' on the terminal and go to Postman to test out the endpoints in the README. To perform the tests run nosetest.

__Relevant pivotal tracker stories.__
#162527329
